src/pageObject.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
 class PageObject {
+  constructor(url) {
+    this.url = url;
+    this.header = this.header;
+    this.header = new Header();
+  }
 
-};
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
+}
 
 class ArticlePage extends PageObject {
+  constructor(slag, url) {
```

Comment:
The parameters in the constructor of the 'ArticlePage' class are not in the correct order as per the task description. It should be (url, slug), not (slug, url).

Reply:
fixed